src/configure/clients/devOps/azureDevOpsClient.ts

```diff
@@ -22,12 +24,62 @@ export class AzureDevOpsClient {
         return this.serviceClient.sendRequest(urlBasedRequestPrepareOptions);
     }
 
-    public async listOrganizations(): Promise<Organization[]> {
-        if (this.organizationMap || this.listOrgPromise) {
-            return this.organizationMap ? this.organizationMap : this.listOrgPromise;
-        }
+    public async createOrganization(organizationName: string): Promise<any> {
+        return this.serviceClient.sendRequest<any>(<UrlBasedRequestPrepareOptions>{
+            url: "https://app.vsaex.visualstudio.com/_apis/HostAcquisition/collections",
+            headers: {
+                "Content-Type": "application/json"
+            },
+            method: "POST",
+            queryParameters: {
+                "collectionName": organizationName,
+                "api-version": "4.0-preview.1",
+                "preferredRegion": "CUS"
+            },
+            body: {
+                "VisualStudio.Services.HostResolution.UseCodexDomainForHostCreation": "true"
```

Comment:
Is this still required or is it the default now? Can you check

Reply:
Not required. Removing it.